docs/_includes/default.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <title>{{ page.title }}</title>
+    <link
+      rel="preload"
+      href="https://opencollective.com/static/images/user.svg"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="prefetch"
+      href="https://www.google-analytics.com/analytics.js"
+      crossorigin="anonymous"
+    />
+    <link rel="preconnect" href="https://stats.g.doubleclick.net" />
```

Comment:
Why is this here?

Reply:
"stats.g.doubleclick.net" Is one of DoubleClick subdomains used for loading tracking pixels or as an alternative for loading Google Analytics, collecting statistics and data that google uses to analyze and display advertisments.

https://tonnygaric.com/blog/prevent-google-analytics-from-making-requests-to-stats-g-doubleclick-net

Reply:
no idea, but that can be removed.

we could also drop GA for something like Matomo.  It's €9/mo which we could pay for out of the fund.  I use it on my personal site and it's bomb.